wix/main.wxs

```diff
@@ -0,0 +1,222 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<!--
+  Copyright (C) 2017 Christopher R. Field.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<!--
+  Please do not remove these pre-processor If-Else blocks. These are used with
+  the `cargo wix` subcommand to automatically determine the installation
+  destination for 32-bit versus 64-bit installers. Removal of these lines will
+  cause installation errors.
+-->
+<?if $(var.Platform) = x64 ?>
+    <?define Win64 = "yes" ?>
+    <?define PlatformProgramFilesFolder = "ProgramFiles64Folder" ?>
+<?else ?>
+  <?define Win64 = "no" ?>
+  <?define PlatformProgramFilesFolder = "ProgramFilesFolder" ?>
+<?endif ?>
+
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'>
+
+    <Product
+        Id='*'
+        Name='Notion'
+        UpgradeCode='2AC3458D-A24D-4828-AF47-013B30999B54'

```

Comment:
Is there a reason you changed the `UpgradeCode` from `42A39E14-335A-4464-AA37-17FDA38FA377`? 
It might be not too important in this early stage of the project, but if you change the upgrade code Windows won't be able to detect the old version (and remove it). The `UpgradeCode` is the single most important id in the whole file.

Reply:
I believe this file has always had the same `UpgradeCode`, the other one was from the previous installer that I don't believe anyone used, since it hadn't been supported in a while. I think at this stage it probably isn't an issue, but there's also no real reason to _not_ use the old `UpgradeCode`, so I'll swap it in.